sphinxcontrib/apidoc/__init__.py

```diff
@@ -28,5 +28,6 @@ def setup(app):
     app.add_config_value('apidoc_module_dir', None, 'env', [str])
     app.add_config_value('apidoc_output_dir', 'api', 'env', [str])
     app.add_config_value('apidoc_excluded_paths', [], 'env', [[str]])
+    app.add_config_value('apidoc_separate_modules', False, 'env', [[bool]])
```

Comment:
s/[[bool]]/[bool]/

(this is a type check, and what's there means "list of bools" when I guess we just want a single bool)

Reply:
Went ahead and fixed this. Also added some tests along the way.